From: Arnaud Ferraris <aferraris@debian.org>
Date: Tue, 16 Dec 2025 10:14:06 +0100
Subject: Makefile: install policies to /usr/share

On Debian, `/usr/lib/firefox/distribution` is a symlink to
`/usr/share/firefox/distribution`, causing piuparts to error out when
installing the `policies.json` file to the former location. Change the
install path to the `/usr/share` folder in order to fix this error.
---
 Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 29df567..e6f29b1 100644
--- a/Makefile
+++ b/Makefile
@@ -3,6 +3,7 @@
 
 DESTDIR :=
 FIREFOX_DIR := /usr/lib/firefox
+FIREFOX_SHARE_DIR := /usr/share/firefox
 MCF_DIR := /usr/lib/mobile-config-firefox
 
 dummy:
@@ -13,7 +14,7 @@ install:
 
 	# Firefox Policies
 	install -Dm644 src/policies.json \
-		"$(DESTDIR)/$(FIREFOX_DIR)/distribution/policies.json"
+		"$(DESTDIR)/$(FIREFOX_SHARE_DIR)/distribution/policies.json"
 
 	# Firefox autoconfig
 	install -Dm644 src/mobile-config-prefs.js \
